shpc/main/modules/templates/singularity.lua

```diff
@@ -71,6 +76,8 @@ if (myShellName() == "bash") then
 {% endfor %}
 end{% endif %}
 
+{% endif %}
```

Comment:
Let's clean up extra white space / newlines if possible!

Reply:
addressed in commit https://github.com/marcodelapierre/singularity-hpc/commit/6466461228b9c2d1d6afd62ace444c863ee3e98f

Reply:
also tested